### PR TITLE
feat: add new api playground to the documentation

### PR DESCRIPTION
**Pull request by enzostvs**
Hello @Wauplin 

I should probably upload the image to the hugging face datasets, as the others images, right ?
Wanted to wait your confirmation first

thx

### CODE REVIEW

**Comment by Wauplin** [comment]
> I should probably upload the image to the hugging face datasets, as the others images, right ?

Yes exactly! Thanks for taking care of it :)

File: docs/hub/api.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ We have open endpoints that you can use to retrieve information from the Hub as
 
 The base URL for those endpoints below is `https://huggingface.co`. For example, to construct the `/api/models` call below, one can call the URL [https://huggingface.co/api/models](https://huggingface.co/api/models)
 
+## The Hub API Playground
+
+Want to try out our API?
+Try it out now on our [Playground](https://huggingface.co/spaces/enzostvs/hub-api-playground)!
+
+<div class="flex justify-center">
+<img class="w-full object-contain" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/billing.png"/>
```
Comment (Wauplin) [inline docs/hub/api.md:13]: Let's not forget to update the link here :)

**Review by Wauplin — COMMENTED** [review]
Looks good to me, thanks @enzostvs. Let's update the screenshot and that's it. As you guessed, you should add the screenshot to this dataset (https://huggingface.co/datasets/huggingface/documentation-images/tree/main/hub) and reference it here.

FYI, here is how the docs looks like for your PR: https://moon-ci-docs.huggingface.co/docs/hub/pr_1073/en/api.

**Review by Wauplin — COMMENTED** [review]
Added a small comment about naming.


Also while reviewing the Playground I realized that we are documenting a `config` parameter for the `/api/spaces` endpoint which doesn't exist. If passed to the endpoint it is simply ignored. The `config` parameter is only available for `/api/models`. Could you:
~1. remove it from [api.md](https://github.com/huggingface/hub-docs/blob/1f36c9e88813e44f5fd169e50075c0ba712ab508/docs/hub/api.md?plain=1#L118) (in this PR?)~. **EDIT:** done as part of https://github.com/huggingface/hub-docs/pull/1075.
2. remove it from the playground as well

Thanks in advance and sorry for the confusion!

**Comment by julien-c** [comment]
(related a bit to #1075)

File: docs/hub/api.md (hunk 1 above)
Comment (enzostvs) [reply docs/hub/api.md]: Done!

File: docs/hub/api.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ We have open endpoints that you can use to retrieve information from the Hub as
 
 The base URL for those endpoints below is `https://huggingface.co`. For example, to construct the `/api/models` call below, one can call the URL [https://huggingface.co/api/models](https://huggingface.co/api/models)
 
+## The Hub API Playground
```
[the remark this answers is not in the document]
Comment (enzostvs) [reply docs/hub/api.md:7]: As discussed in Slack, we'll keep Hub API Playground for now
Everything is normally called with the same name, but subject to change in the coming days ~

**Pull request merged**